src/libpmemfile/path_resolve.c

```diff
@@ -110,7 +110,7 @@ resolve_symlink(struct resolved_path *result,
 		}
 	}
 
-	if (! *is_last_component)
+	if (!(*is_last_component))
```

Comment:
I don't see any point in doing that. If anything can be improved here it's space removal.